pkg/changelog/changelog.go

```diff
@@ -89,7 +89,7 @@ func (c *changelog) Tail() *entry.Entry {
 }
 
 // NewChangelog creates a new changelog datastructure.
-func NewChangelog(repoName string, repoOwner string) Changelog {
+func NewChangelog(repoOwner string, repoName string) Changelog {
```

Comment:
Fixing the correct order of params https://github.com/chelnak/gh-changelog/blob/main/pkg/parser/parser.go#L98

Reply:
Nice catch!